src/prisma/schema.prisma

```diff
@@ -1122,6 +1134,41 @@ model ProposalReviewer {
   @@index([userId])
 }
 
+enum ProposalRubricCriteriaType {
+  range
+}
+
+model ProposalRubricCriteria {
+  id          String   @id   @db.Uuid @default(uuid())
+  title       String
+  description String?
+  proposalId  String      @db.Uuid
+  proposal    Proposal @relation(fields: [proposalId], references: [id], onDelete: Cascade)
+  type        ProposalRubricCriteriaType
+  parameters  Json
+  answers     ProposalRubricCriteriaAnswer[]
+
+  @@unique([title, proposalId])
```

Comment:
Make sure criteria don't overlap

Reply:
@motechFR can we actually remove this. there's no technical need to enforce it - i'd rather we put rules like that in the UI instead.